### PR TITLE
feat(contributing): add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by saugereau**
A proposition of CONTRIBUTING.md that can be discussed together

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+[Styleguides](#styleguides)
+  * [Git Commit Messages](#git-commit-messages)
+  * [Java Styleguide](#java-styleguide)
+  * [Kotlin Styleguide](#kotlin-styleguide)
+
+## How Can I Contribute?
+
+### Reporting Bugs
+
+This section guides you through submitting a bug report for Atom. Following these guidelines helps maintainers and the community understand your report :pencil:, reproduce the behavior :computer: :computer:, and find related reports :mag_right:.
```
Comment (clemstoquart) [inline CONTRIBUTING.md]: `for Atom` I think JCV would be better here :)

**Review by clemstoquart — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (saugereau) [reply CONTRIBUTING.md]: oupss ...my bad ...
Fixed

**Review by clemstoquart — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+2. Follow the [styleguides](#styleguides)
+3. After you submit your pull request, verify that travis build (can be seen in your PR status)
+
+## Styleguides
+
+### Git Commit Messages
+
+As git commit messages are used to generate changelog, it can be usefull to have the same formalism.
+We suggest to use [Angular Git commit guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines)
```
Comment (leomillon) [inline CONTRIBUTING.md]: Or maybe https://www.conventionalcommits.org ?

Comment (Crow-EH) [reply CONTRIBUTING.md]: It's more precise and based on Angular anyway. :+1: 

Here the commit linter they recommend if you want more details : https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional

**Pull request merged**